nemo/lightning/nemo_logger.py

```diff
@@ -202,10 +202,13 @@ def _setup_trainer_model_checkpoint(self, trainer, log_dir, ckpt=None):
                 ModelCheckpoint.CHECKPOINT_NAME_LAST = callback.filename + '-last'
 
     def _handle_task_config(self, task_config, log_dir):
-        task_config.save_config_img(log_dir / "task.png")
-        task_json = serialization.dump_json(task_config)
-        with open(log_dir / "task.json", "w") as f:
-            f.write(task_json)
+        try:
+            task_config.save_config_img(log_dir / "task.png")
+            task_json = serialization.dump_json(task_config)
```

Comment:
Can you move `from fiddle._src.experimental import serialization` inside this try except too?